test.md

```diff
@@ -0,0 +1 @@
+this will be ignored by git
```

Comment:
- [ ] Please make sure that this file is ignored by git, as this is not supposed to be in your repository.